test_data.rb

```diff
@@ -491,6 +491,21 @@ def test_main_station_hints
     end
   end
 
+  def test_normalised_code
+    STATIONS.each do |row|
+      if (row["country"] != 'GB' )
+        assert !row["normalised_code"].nil?, "Station #{row["name"]} (#{row["id"]}) does not have a normalised_code"
+        expected_id = row["id"]
+        if (!row["same_as"].nil?)
+          expected_id = row["same_as"]
+        end
```

Comment:
`expected_id = row["same_as"] || row["id"]` seems clearer to me

Reply:
Done. Thanks. I'm completely new to Ruby